### ISSUE TITLE
Has no dependency to query

### ISSUE THREAD

**Issue by mbaev** [mwlang/gentelella-rails#4]
After instal the gem, run `bundler install` and visit to front page of my app, I see next error:

```
couldn't find file 'jquery' with type 'application/javascript'
Checked in these paths: 
...
```
In dependencies of the gem I don't see any requirements to jquery:
```Ruby
# .../gems/gentelella-rails-0.1.3/lib/gentelella-rails.rb
require 'bootstrap-sass'
require 'font-awesome-sass'
require 'nprogress-rails'
require 'icheck-rails'
require 'momentjs-rails'
require 'bootstrap-daterangepicker-rails'
require 'fastclick-rails'
require 'chart-js-rails'
require 'flot-rails'
require 'datejs-rails'
require 'switchery-rails'
require 'select2-rails'
require 'parsley-rails'
```

But I see requirements of jQuery in manifest
```Coffee
# .../gems/gentelella-rails-0.1.3/assets/javascripts/gentelella.coffee
#= require jquery
#= require jquery_ujs
```
I think you need to add requirement
```Ruby
gem 'jquery-rails'
```
to your `gentelella-rails.rb`

**Comment by mwlang** [comment]
what version of ruby and rails?

**Comment by mbaev** [comment]
```
$ rails -v
Rails 5.0.2

$ ruby -v
ruby 2.3.1p112 (2016-04-26 revision 54768) [x64-mingw32]
```

**Comment by mwlang** [comment]
I couldn't replicate, but I also don't see the harm in requiring explicitly.  So I've added the require and have released v0.1.4.  Try ```bundle update``` and ensure you have 0.1.4 of the gem and give it another whirl.

**Comment by mbaev** [comment]
```
$ bundler update gentelella-rails
Fetching gem metadata from https://rubygems.org/.......
Fetching version metadata from https://rubygems.org/..
Fetching dependency metadata from https://rubygems.org/.
Resolving dependencies....
Using rake 12.0.0
...
Installing gentelella-rails 0.1.4 (was 0.1.3)
```
After restarting server it works. Thanks.
